content/shmem_broadcast.tex

```diff
@@ -72,12 +72,12 @@
     needed to ensure this): The \VAR{pSync} array on all \ac{PE}s in the
     \activeset{} is not still in use from a prior call to a broadcast routine.  The
     \dest{} array on all \ac{PE}s in the \activeset{} is ready to accept the
-    broadcast data.
-    
+    broadcast data. 
+       
```

Comment:
Could you push a commit to remove this whitespace change?  This is a "document edit", so it should not impact your special ballot.